app/src/main/kotlin/com/hedvig/app/feature/loggedin/ui/LoggedInViewModel.kt

```diff
@@ -68,6 +72,17 @@ class LoggedInViewModelImpl(
         .map { it == 3 }
         .collect(_shouldOpenReviewDialog::tryEmit)
     }
+
+    viewModelScope.launch {
+      when (val result = memberIdUseCase.memberId()) {
+        is GetMemberIdUseCase.MemberIdResult.Error -> {
+          slimber.log.e { "Could not identify member to customer.io - could not fetch member id" }
+        }
+        is GetMemberIdUseCase.MemberIdResult.Success -> {
+          CustomerIO.instance().identify(identifier = result.id, attributes = mapOf("os" to "android"))
+        }
+      }
+    }
```

Comment:
For getting the memberID, there's a new way apprently, which we just started using. Basically take a look at this https://github.com/HedvigInsurance/android/blob/6c86b2a6f988600effeca786a0a3f63a3bdb910c/datadog/src/main/kotlin/com/hedvig/android/datadog/memberid/DatadogMemberIdUpdatingAuthEventListener.kt#L30-L48 and then take a look at this discussion https://hedviginsurance.slack.com/archives/C03DTURB621/p1675757933975849

The gist is that the new auth tokens we get do properly handle embedding the member ID in there, so you can fetch it locally from the stored token locally. 

Maybe this is an opportunity to also have this customer-io functionality also hook into the `AuthEventListener` events, and do this on login, and clear the token on logout? You can check `DatadogMemberIdUpdatingAuthEventListener` for how we do that for Datadog.

Reply:
And would need to extract that `extractMemberIdFromAccessToken` to somewhere else. Didn't do it before since it didn't need to be used elsewhere, but now we can think of where it can fit. core-common-andoird is the first candidate, as it's a simple function of type `(String) -> String?`, but it does have an android dependency since it's using `android.util.Base64`.

Reply:
Thats great, I was looking for something like this :) Will add a new PR with these changes together with a new module 👍